template/src/components/Controls.native.tsx

```diff
@@ -29,9 +22,13 @@ import {
 } from '../../agora-rn-uikit';
 import Recording from '../subComponents/Recording';
 import LiveStreamControls from './livestream/views/LiveStreamControls';
+import {useVideoCall} from 'fpe-api';
```

Comment:
Minor:  if we want to use it like this, then do we wanna call it fpe-api ? from what i understand fpe-api are fpe facing and not to be used inside the app, we should not import from fpe-api but from the actual implementation file.

Reply:
done